Tests/PKHeX.Core.Tests/Simulator/ShowdownSetTests.cs

```diff
@@ -184,7 +175,8 @@ public void TestGenerate()
                 foreach (var e in ez)
                 {
                     var la = new LegalityAnalysis(e);
-                    Assert.IsTrue(la.Valid);
+                    la.Valid.Should().BeTrue($"Because all generated Pokemon for {i:000} should be valid");
```

Comment:
This is the part of the test is failing, where i=12

Reply:
I had left it as a debug way test IEncounterable -> pkm code back a while ago.

I had commented out (see `//[TestCategory(SimulatorParse)]`) thus the test was never part of the usual test suite. With the recent addition of shadow lock checks, it's likely failing because of that.

Reply:
> I had commented out (see //[TestCategory(SimulatorParse)]) 

Ah, right, I think I remember seeing something like that.

> Is it possible to just skip this method for testing like before?

Yep. I can either use the Skip attribute or just comment it out. I'll decide which to use when I push an update